### PR TITLE
#261 Add copy to clipboard functionality to Raw Response

### PR DESCRIPTION
**Pull request by micmro**
Screen Shot 2019-10-06 at 8 17 31 PM

### CODE REVIEW

**Comment by soulgalore** [comment]
Looks great but what bout moving it to content folder:
PerfCascade - HAR Waterfall viewer

That way we copy/paset just the HTML/content and that makes it easier when comparing old data.

**Comment by micmro** [comment]
Ah, ok that was the one you meant 😄. For some reason I though about the raw data. What do you think of showing it in both? Copying the raw data is quite a pain too.

File: src/ts/waterfall/details-overlay/svg-details-overlay.ts [hunk 1]
```diff
@@ -51,10 +65,14 @@ export function createRowInfoOverlay(overlay: OpenOverlay, y: number, detailsHei
...
   });
 
+  const body = createDetailsBody(requestID, detailsHeight, overlay.entry);
   const closeBtn = createCloseButtonSvg(y);
-  closeBtn.addEventListener("click", () => overlay.onClose(overlay.index));
+  closeBtn.addEventListener("click", () => {
+    overlay.onClose(overlay.index);
+    body.removeEventListener("click", onTabDataCopyClick);
+  });
+  body.addEventListener("click", onTabDataCopyClick);
```
Comment (micmro) [inline src/ts/waterfall/details-overlay/svg-details-overlay.ts:74]: use event delegation as we can't bind callbacks before the tab has been added to the DOM, this also allows us to clean up the event listeners when it gets removed.

**Comment by soulgalore** [comment]
Both sounds great :)

**Review by soulgalore — APPROVED** [review]
Woho!!

**Comment by micmro** [comment]
Released with `2.7.0`

**Pull request merged**